components/dashboard/src/settings/GitIntegrations.tsx

```diff
@@ -1,9 +1,9 @@
 import { SettingsPage } from "../components/SettingsPage";
 import settingsMenu from './settings-menu';
 
-export default function GitIntegration() {
+export default function GitIntegrations() {
     return <div>
-        <SettingsPage title='Settings' subtitle='Manage integration with your Git hosters' menuEntries={settingsMenu}>
+        <SettingsPage title='Git Integrations' subtitle='Manage integration with your Git hosters' menuEntries={settingsMenu}>
```

Comment:
**thought**: I still think we could go with _Intergrations_ here and with Variables in [above](https://github.com/gitpod-io/gitpod/pull/3456/files#diff-4b2875688f8e0695a02fc72e9b4442a37ddd187e8c7ac3aec99305ea540313ccR8) but we can change these later this week or next week if this sounds good.